api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddMemberRoleToPersonGenerator.java

```diff
@@ -40,7 +40,7 @@ String getRoleType() {
 	}
 
 
-	FieldOptions getRoleActivityFieldOptions(VitroRequest vreq) throws Exception {
+    FieldOptions getRoleActivityFieldOptions(VitroRequest vreq) throws Exception {
```

Comment:
I believe this line is a part of Michel's functional update... and can likely be removed from this pull-request.